tests/llmcompressor/transformers/compression/test_run_compressed.py

```diff
@@ -22,12 +21,21 @@ class TestQuantizationMatches(unittest.TestCase):
     def setUpClass(cls):
         cls.test_dir = tempfile.mkdtemp()
 
-        cls.compressed_model = SparseAutoModelForCausalLM.from_pretrained(
-            cls.model_stub, torch_dtype="auto", device_map="auto", run_compressed=True
+        # TODO: Give option on HFQuantizer to run run_compressed True/False
```

Comment:
I would just skip this test for now, then reenable when run_compressed is fully supported